### PR TITLE
Added support for get payment lists (CS1/CS2)

### PR DESCRIPTION
**Pull request by armandojaleo**
## Modifications
Added support for get payment lists (CS1/CS2)

## API Reference
CS1: https://api-reference.checkout.com/previous/#operation/getPayments
CS2: https://api-reference.checkout.com/#tag/Payments/paths/~1payments/get

### CODE REVIEW

File: test/payments_request_test.go [hunk 1]
```diff
@@ -170,6 +120,30 @@ func TestMakeCardVerification(t *testing.T) {
 
 }
 
+func TestRequestPaymentList(t *testing.T) {
```
Comment (Armando-Ibarra) [inline test/payments_request_test.go:123]: In the future we should avoid using the old way of testing, and refactor to our current pattern

**Review by Armando-Ibarra — APPROVED** [review]
very good job

File: payments/abc/client.go [hunk 2]
```diff
@@ -40,6 +40,26 @@ func (c *Client) RequestPayment(request PaymentRequest, idempotencyKey *string)
 	return &response, nil
 }
 
+func (c *Client) RequestPaymentList(request payments.QueryRequest) (*GetPaymentListResponse, error) {
+	auth, err := c.configuration.Credentials.GetAuthorization(configuration.SecretKeyOrOauth)
```
Comment (martinseco) [inline payments/abc/client.go:44]: This should be SecretKey only. There is no OAuth in previous.

**Review by martinseco — APPROVED** [review]
Looks good!

**Pull request merged**